### PR TITLE
Feature/87756 add additional information fields for Academy Transfers

### PR DESCRIPTION
**Pull request by acarbutt**
Add additional information fields to the Academies level.
POST, PATCH and GET updated with the new fields
Data copied to academy level from project (Project additional information can be removed post Transfers deployment)

### CODE REVIEW

File: TramsDataApi.Test/Integration/AcademyTransferProjectIntegrationTests.cs [hunk 1]
```diff
@@ -362,7 +362,8 @@ public async Task CanUpdateTheTransferringAcademiesOfAnAcademyTransferProject()
                     (List<TransferringAcademiesRequest>) Builder<TransferringAcademiesRequest>
                         .CreateListOfSize(2).All()
                         .With(ta => ta.IncomingTrustUkprn = randomGenerator.NextString(8, 8))
-                        .With(ta => ta.OutgoingAcademyUkprn = randomGenerator.NextString(8, 8)).Build())
+                        .With(ta => ta.OutgoingAcademyUkprn = randomGenerator.NextString(8, 8))
```
Comment (StewartBarker) [inline TramsDataApi.Test/Integration/AcademyTransferProjectIntegrationTests.cs:365]: There are a few tests in this file that don't include the new fields, not sure if they are 100% necessary but I'd just double check

File: TramsDataApi/Migrations/TramsDb/TramsDbContextModelSnapshot.cs [hunk 2]
```diff
@@ -583,6 +583,21 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("A2BSchoolLoan","sdd");
                 });
 
+            modelBuilder.Entity("TramsDataApi.DatabaseModels.A2BSelectOption", b =>
```
Comment (StewartBarker) [inline TramsDataApi/Migrations/TramsDb/TramsDbContextModelSnapshot.cs:586]: I know this file is probably auto generated but just flagging this to check if it is intentional as looks like A2B

**Review by StewartBarker — COMMENTED** [review]
Looks good, just potentially some missing tests and might be best double checking that migration as looks like it includes an A2B table.

File: TramsDataApi.Test/Integration/AcademyTransferProjectIntegrationTests.cs (hunk 1 above)
Comment (acarbutt) [reply TramsDataApi.Test/Integration/AcademyTransferProjectIntegrationTests.cs:365]: Updated all unit tests with the fields

**Review by StewartBarker — APPROVED** [review]

**Pull request merged**